beacon-chain/rpc/beacon_chain_server.go

```diff
@@ -72,3 +73,39 @@ func (bs *BeaconChainServer) ListValidatorBalances(
 	}
 	return &ethpb.ValidatorBalances{Balances: res}, nil
 }
+
+// GetValidators retrieves the active validators with an optional historical epoch flag to
+// to retrieve validator set in time.
+//
+// TODO(#3045): Implement validator set for a specific epoch. Current implementation returns latest set,
+// this is blocked by DB refactor.
+func (bs *BeaconChainServer) GetValidators(
+	ctx context.Context,
+	req *ethpb.GetValidatorsRequest) (*ethpb.Validators, error) {
+
+	validators, err := bs.beaconDB.Validators(ctx)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "could not retrieve validators: %v", err)
+	}
+
+	pageSize := int(req.PageSize)
+
+	start, err := strconv.Atoi(req.PageToken)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "could not get page token: %v", err)
+	}
+	end := start + pageSize
+	nextPage := end + 1
+	if end > len(validators) {
+		end = len(validators)
+		pageSize = end - start
+		nextPage = 0
+	}
+
+	res := &ethpb.Validators{
+			Validators: validators[start:end],
+			TotalSize: int32(pageSize),
+			NextPageToken: string(nextPage),
```

Comment:
Are you sure this does what you expect? Please add unit tests that have the full expected response.

Example: int(55) becomes "7" 
https://play.golang.org/p/idfXG1F5upp